CONTRIBUTING.md

```diff
@@ -1,13 +1,13 @@
 # Contributing
 
-[![Gitter chat][gitter-image]][gitter-url]
+[![Matrix chat][matrix-image]][matrix-url]
 [![Merged PRs][prs-merged-image]][prs-merged-url]
 [![GitHub contributors][contributors-image]][contributors-url]
 [![CLA assistant][cla-assistant-image]][cla-assistant-url]
 [![license][license-image]][license-url]
 
-[gitter-url]: https://gitter.im/tldr-pages/tldr
-[gitter-image]: https://img.shields.io/badge/chat-on_gitter-deeppink
+[matrix-url]: https://matrix.to/#/#tldr-pages:matrix.org
+[matrix-image]: https://img.shields.io/badge/chat-on_matrix-deeppink
```

Comment:
Sadly (:sob:) the Gitter UI is no more, so it doesn't make sense to have the deep pink badge color anymore. Maybe something closer to Element's branding would make more sense here?

```suggestion
[matrix-image]: https://img.shields.io/badge/chat-on_matrix-lightseagreen
```

Reply:
Or maybe [something](https://img.shields.io/badge/chat-on_[matrix]-white) more generically related to Matrix? ![Screen Shot 2023-02-27 at 16 04 02](https://user-images.githubusercontent.com/478237/221615440-d7797709-d42f-4861-84bf-d233955968b7.png)

Reply:
Applied the green suggestion, if required I will update it to anything else.

Reply:
Thoughts on using the shields.io badge for this?

![](https://img.shields.io/matrix/tldr-pages:matrix.org)

```md
https://img.shields.io/matrix/tldr-pages:matrix.org
```

(Feel free to restyle it using the query params provided, but just so we have the user count with it.)

Reply:
The native badge is better indeed, and I agree with using the more explicit "Chat on Matrix" custom label. Nice! Marking this thread as resolved.